src/main/java/org/ow2/proactive/connector/iaas/rest/KeyPairRest.java

```diff
@@ -64,8 +64,8 @@
     @Produces("application/json")
     @Path("{infrastructureId}/keypairs")
     public Response createKeyPair(@PathParam("infrastructureId") String infrastructureId, final String instanceJson) {
-        log.info("Receive create request for infrastructure " + infrastructureId + " with parameter " + instanceJson);
         Instance instance = JacksonUtil.convertFromJson(instanceJson, Instance.class);
+        log.info("Receive create request for infrastructure " + infrastructureId + " with parameter " + instance);
```

Comment:
same

Reply:
Leaving in small cap, just as other id in the logs